models/topics-models.js

```diff
@@ -16,8 +22,12 @@ exports.selectArticleById = (article_id) => {
     })
 }
 
-exports.selectArticles = () => {
-    return db.query('SELECT articles.article_id, articles.author, title, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC;').then((result) => {
+exports.selectCommentsById = (article_id) => {
+    return db.query('SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;', [article_id])
+    .then((result) => {
+        if(result.rows.length === 0) {
```

Comment:
We need to check specifically why we are returning an empty row. 
This could be due to two reasons...
article exists but has no comments.
OR
article does not exist.
You will need to check whether the article exists in order to determine this exactly. If it has no comments, you want to return an empty array, if it doesn't exist, we want to throw an error. We should also test for both scenarios